sequencer/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 pub mod api;
 mod block;
-mod block2;
+pub mod block2;
```

Comment:
Why `pub` here? I note that `block` is not `pub` and my default is to mimic `block` in `block2`.

Reply:
yes, good point